Bio/bgzf.py

```diff
@@ -770,8 +769,10 @@ def __init__(self, filename=None, mode="w", fileobj=None, compresslevel=6):
             if "w" not in mode.lower() and "a" not in mode.lower():
                 raise ValueError("Must use write or append mode, not %r" % mode)
             if "a" in mode.lower():
-                raise NotImplementedError("Append mode is not implemented yet")
-                # handle = _open(filename, "ab")
+                handle = _open(filename, "ab")
+                # py27 on Windows will get confused about our current position
```

Comment:
Python 2.7 is not supported any more.

Reply:
Yes, sadly this pull request has languished - I've have less time available for Biopython that I used to.

Reply:
So, the question now is do we still need this ``handle.seek(0, os.SEEK_END)`` on Windows under Python 3.6 onwards?

Guess we should try it and see... will need another test